packages/babel-core/test/__snapshots__/config-loading.js.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`@babel/core config loading programmatic plugins/presets should throw when plugin contains \`enter\` handler 1`] = `"[BABEL] /Users/jh/code/babel/packages/babel-core/test/fixtures/config-loading/folder/example.js: .visitor cannot contain catch-all \\"enter\\" or \\"exit\\" handlers. Please target individual nodes."`;
```

Comment:
Just realize that since `.enter` and `.exit` are not allowed in `visitors` since v7, we could possibly remove https://github.com/babel/babel/blob/07347544fda9c88064379ec705a3c42224aa4c0a/packages/babel-traverse/src/context.js#L27

Reply:
Is it currently possible to use them if calling `traverse` directly?

Reply:
yeah I think so, it was added to core, although we could revisit that (change it back) too

Reply:
Yeah it is possible to run `require("@babel/traverse").traverse({ enter() {} })`. Guess we should also remove that in Babel 8, to be aligned with what `@babel/core` limits on plugins.

Reply:
I think that `enter` for advanced users (= people using `traverse` directly) could be useful, and also it's the default behavior of all the other traversal methods.